packages/react-native-editor/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ For each user feature we should also add a importance categorization label  to i
 -->
 
 ## Unreleased
+
+## 1.97.0
 -   [*] [internal] Upgrade compile and target sdk version to Android API 33 [#50731]
```

Comment:
It appears we may have missed an entry for the change log. The [iOS dictation fix](https://github.com/WordPress/gutenberg/blob/e576cddb27d6d92bafcb41b1cdfdd2e288083c64/packages/react-native-editor/CHANGELOG.md?plain=1#L26) that merged in https://github.com/WordPress/gutenberg/pull/49452 appears to be placed on an erroneous line. This is likely caused by an inaccurate merge conflict resolution. 

Will you relocate this entry to 1.97.0 please?